environment.yml

```diff
@@ -10,6 +10,8 @@ dependencies:
   - toml=0.9.4
   - h5py=2.9.0
   - scipy=1.4.1
+  - pyyaml=5.3.1
+  - semver=2.9.0
```

Comment:
any reason for this version rather than the latest?

Reply:
I wrote this in manually, and just put in the version that I got from a conda install. Is there a newer conda version? I have no idea whether the version schemes are all consistent.

Reply:
I think this explicitly specifies the latest versions current on conda, if that helps?

Reply:
Apparently there is a newer version on a different conda channel, but I've no idea whether using a different channel is a good idea. @bobturneruk, @aflag ?

Reply:
I think it depends - do we want features / bug fixes from the newer version? Is it on a channel we trust?

Reply:
I think having something like pyyaml>=5.3 for a library should be fine. In the actual application that uses the library we'd want to have some sort of pinning (the spec-file.txt net work sim's case, for instace.

Reply:
BTW, can you rebase master onto your branch, it'd be nice to also update the meta.yaml file